microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,163 @@
-
-##### [.NET/C#](#tab/dotnetcsharp)
-
-* [CoreWebView2Controller.DefaultBackgroundColor Property](/dotnet/api/microsoft.web.webview2.core.corewebview2controller.defaultbackgroundcolor)
-  * [CoreWebView2Color Struct](/dotnet/api/microsoft.web.webview2.core.corewebview2color)
-
-##### [WinRT/C#](#tab/winrtcsharp)
-
-* [CoreWebView2Controller.DefaultBackgroundColor Property](/dotnet/api/microsoft.web.webview2.core.corewebview2controller.defaultbackgroundcolor#microsoft-web-webview2-core-corewebview2controller-defaultbackgroundcolor)
-  * [CoreWebView2Color Struct](/dotnet/api/microsoft.web.webview2.core.corewebview2color)
-
-##### [Win32/C++](#tab/win32cpp)
-
-* [ICoreWebView2Controller::DefaultBackgroundColor property (get](/microsoft-edge/webview2/reference/win32/icorewebview2controller2#get_defaultbackgroundcolor), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2controller2#put_defaultbackgroundcolor)
-  * [COREWEBVIEW2_COLOR struct](/microsoft-edge/webview2/reference/win32/icorewebview2#corewebview2_color)
+<!-- ------------------------------ -->
+#### APIs for Window to Visual hosting
 
----
+For a list of APIs that can be used when configuring WebView2 Window to Visual hosting (or for Windowed hosting), see [Rendering WebView2 in non-framework apps](./overview-features-apis.md#rendering-webview2-in-non-framework-apps) in _Overview of WebView2 features and APIs_.
 
 
 <!-- ====================================================================== -->
-## Visual hosting
+## Visual hosting: For more granular control over layout
 
-In visual hosting, content is embedded to a given location on the application. This location must handle how content will scale and behave in an app when the user interacts with the application.  In addition to the window management described for windowed hosting, visual hosting will need the app to manage the composition-based rendering, when it receives any user interactions.  For more information about visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
+When using _Visual hosting_, your host app takes spatial input (such as mouse or touch input) from the user.  Your app sends this input to the WebView2 control.
 
-If your WebView2 app uses visual hosting:
+In Visual hosting, content is embedded to a given location on the application. This location must handle how content will scale and behave in an app when the user interacts with the application.  In addition to the window management described for Windowed hosting, Visual hosting will need the app to manage the composition-based rendering, when it receives any user interactions.  For more information about Visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
```

Comment:
```suggestion
In Visual hosting, content is embedded at a given location in the application. This location must handle how content will scale and behave when the user interacts with the application.  In addition to the window management described for Windowed hosting, Visual hosting requires the app to manage the composition-based rendering when it receives any user interactions.  For more information about Visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
```

Reply:
This section is a bit confusing to me. 

"This location must handle how content will scale and behave in an app when the user interacts with the application." - What content is this referring to, the WebView2 content? What does it mean to scale when the user interacts with the application (like a gesture)?

"the app to manage the composition-based rendering when it receives any user interactions." - What does it mean to "manage the ... rendering"? Which user interactions would cause this, or what are the scenarios?

Reply:
Closing - safe to close this, b/c entered hidden todo comments (local VS Code, not committed/pushed yet).  prevents "Outdated", confusing Comments.  after push, will create fresh Comment at "todo".

@johna-ms @champnic re: process to handle Comments and not have Outdated comments after push commit that modifies line.  "Outdated" comments are hard to work w/ - misrepresentative surrounding Markdown.